code/game/objects/items/RCD.dm

```diff
@@ -29,24 +29,32 @@ RLD
 	var/datum/effect_system/spark_spread/spark_system
 	var/matter = 0
 	var/max_matter = 100
-	var/sheetmultiplier	= 4 //Controls the amount of matter added for each glass/iron sheet, triple for plasteel
-	var/plasteelmultiplier = 3 //Plasteel is worth 3 times more than glass or iron
+	var/sheetmultiplier	= 4 //Controls the amount of matter added for each glass/metal sheet, triple for plasteel
```

Comment:
All the metal changes need to be changed to iron.

Reply:
in the comments

Reply:
Replaced all metal with iron.